### DISCUSSION TITLE
Future Ready Talent Perks

### DISCUSSION THREAD

**Discussion by ghost** [github/india#131] [Students]
How to access the perks made available by the Future Ready Talent program?
1.GitHub Student Developer Pack. (Do we need to send the request by keeping future-ready talent as our organization?)

Please Help...

**Comment by scmishra-cse** [comment]
> How to access the perks made available by the Future Ready Talent program? 1.GitHub Student Developer Pack. (Do we need to send the request by keeping future-ready talent as our organization?)
> 
> Please Help...

Hi, @s-Alpha-k As I know you should need to enter your real valid organization with college email I'd and some proof like a College identity card. After some time you will receive your GitHub Student Developer Pack (Correct me If I'm wrong)

**Reply by ghost** [reply]
I know that but, why is it mentioned in the perks of the Future-ready talent virtual internship program? I thought they will provide it to every participant of the internship without any proof and all, that's why they have mentioned it over there.
